proto/v1/grafeas.proto

```diff
@@ -208,20 +208,20 @@ message Occurrence {
   // resource.
   oneof details {
     // Describes a security vulnerability.
-    grafeas.v1.vulnerability.Details vulnerability = 8;
+    grafeas.v1.VulnerabilityOccurrence vulnerability = 8;
```

Comment:
Do you think we should we rename all these fields to be [KIND]Occurrence as well?

Reply:
I'd vote for now, for brevity. However, here are suggestions for renaming, to be consistent with the overall naming strategy:
* `derived_image` -> `image`
* `installation` -> `package`
* `discovered` -> `discovery`

Reply:
@vtsao do you mean changing all lines to (for example) `grafeas.v1.VulnerabilityOccurrence vulnerability_occurrence = 8;`?

Reply:
Oops, typo, should have been `I'd vote for no`.

Reply:
Yeah this LGTM.